.github/workflows/terraform.yml

```diff
@@ -40,3 +40,11 @@ jobs:
         id: validate
         if: github.event_name == 'pull_request'
         run: terraform validate
+
+      - name: Update README Files
+        uses: terraform-docs/gh-actions@v1.0.0
+        with:
+          working-dir: "Terraform_modules/${{ matrix.module }}"
+          config-file: "../../.terraform.docs.yml"

```

Comment:
The conventional name for the file is `.terraform-docs.yml`

Reply:
changed